README.md

```diff
@@ -40,7 +42,7 @@ Example Playbook
           nfs_enable:
             server: "{{ inventory_hostname in groups['nfs_server'] }}"
             clients: "{{ inventory_hostname in groups['nfs_clients'] }}"
-          nfs_export: "{{ hostvars['nfs_server']['nfs_export'] }}"
+          nfs_server: "{{ hostvars['nfs_server']['ansible_host'] }}"
```

Comment:
Do we need nfs_export as well as nfs_server?

Reply:
Seems to default `/srv` if omitted

Reply:
Different things: `nfs_export` is what goes in `/etc/exports` on server, `nfs_server` is what goes in `mount` command on client.

Reply:
> Seems to default `/srv` if omitted

Yup `nfs_export` has a default in normal place, so that's ok.

Reply:
As per @jovial's comment the /srv default is in normal place. If you meant a default for `nfs_server` then I don't think there's a way to give a sensible one, unless you want to start having assumptions about group names etc.